apps/server/src/modules/tool/external-tool/mapper/external-tool-request.mapper.ts

```diff
@@ -141,7 +141,12 @@ export class ExternalToolRequestMapper {
 		if (!externalToolMediumParams) {
 			return undefined;
 		}
-		return { ...externalToolMediumParams };
+		return {
```

Comment:
please add a `new ExternalToolMediumDto()` to create an object instance instead of a plain object